extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/handler/WemoLightHandler.java

```diff
@@ -0,0 +1,363 @@
+        logger.debug("WeMoLightHandler disposed.");
+
+        removeSubscription();
+
+        if (refreshJob != null && !refreshJob.isCancelled()) {
+            refreshJob.cancel(true);
+            refreshJob = null;
+        }
+    }
+
+    private synchronized WemoBridgeHandler getWemoBridgeHandler() {
+
+        if (this.wemoBridgeHandler == null) {
+            Bridge bridge = getBridge();
+            if (bridge == null) {
+                logger.error("Required bridge not defined for device {}.", wemoLightID);
+                return null;
+            }
+            ThingHandler handler = bridge.getHandler();
+            if (handler instanceof WemoBridgeHandler) {
+                this.wemoBridgeHandler = (WemoBridgeHandler) handler;
+            } else {
+                logger.debug("No available bridge handler found for {} bridge {} .", wemoLightID, bridge.getUID());
+                return null;
+            }
+        }
+        return this.wemoBridgeHandler;
+    }
+
+    /**
+     * The {@link handleCommand}
+     */
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+
+        Configuration configuration = getConfig();
+        configuration.get(DEVICE_ID);
+
+        WemoBridgeHandler wemoBridge = getWemoBridgeHandler();
+        if (wemoBridge == null) {
+            logger.debug("wemoBridgeHandler not found, cannot handle command");
+            return;
+        }
+        String devUDN = wemoBridge.getThing().getConfiguration().get(ROOT_UDN).toString();
+        logger.trace("WeMo Bridge to send command to : {}", devUDN);
+
+        String value = null;
+        String capability = null;
+        switch (channelUID.getId()) {
+            case CHANNEL_BRIGHTNESS:
+                capability = "10008";
+                if (command instanceof PercentType) {
+                    logger.trace("wemoLight received Value {}", ((PercentType) command).intValue());
+                    int value1 = (((PercentType) command).intValue() * 255 / 100);
+                    value = value1 + ":0";
+                } else if (command instanceof OnOffType) {
+                    switch (command.toString()) {
+                        case "ON":
+                            value = "255:0";
+                            break;
+                        case "OFF":
+                            value = "0:0";
+                            break;
+                    }
+                } else if (command instanceof IncreaseDecreaseType) {
+                    // ToDo : not implemented yet
+                }
+                break;
+            case CHANNEL_STATE:
+                capability = "10006";
+                switch (command.toString()) {
+                    case "ON":
+                        value = "1";
+                        break;
+                    case "OFF":
+                        value = "0";
+                        break;
+                }
+                break;
+        }
```

Comment:
As you can seee, the binding only supports the original WeMo LED lights which have just these two channels. I do not recommend to support other lights cause of general reliability of WeMo devices.